crates/kilt-dip-primitives/src/export/mod.rs

```diff
@@ -16,21 +16,20 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-/// Verification logic to integrate a child chain as a DIP provider.
-pub mod child;
 /// Verification logic to integrate a sibling chain as a DIP provider.
-pub mod sibling;
+pub mod parachain;
+/// Verification logic to integrate a child chain as a DIP provider.
+pub mod relaychain;
 
 mod common;
 
-pub use child::{
-	DipChildProviderStateProofVerifierError, KiltVersionedChildProviderVerifier, VersionedChildParachainDipStateProof,
+pub use parachain::{
+	DipParachainStateProofVerifierError, KiltVersionedParachainVerifier, VersionedDipParachainStateProof,
 };
-pub use sibling::{
-	DipSiblingProviderStateProofVerifierError, KiltVersionedSiblingProviderVerifier,
-	VersionedSiblingParachainDipStateProof,
+pub use relaychain::{
+	DipRelaychainStateProofVerifierError, KiltVersionedRelaychainVerifier, VersionedRelaychainStateProof,
```

Comment:
is this not a bit redundant? The mods are public or not?

Reply:
Perhaps. The goal is to allow a consumer to import directly from the root, and we export the most important types there.

Reply:
I see. I do not have a strong opinion on that. Discard the comment if you like.

Maybe the developer experience is better having shorter names. Instead of `DipParachainStateProofVerifierError` it could be renamed to `StateProofVerifierError`, which is imported from `parachain`? Just an idea.

Reply:
There are conflicts if the same type is re-exported from two different modules. I would move on for now, but won't close this issue. It will probably be part of the next batch of improvements.